src/host/srvinit.cpp

```diff
@@ -256,20 +327,31 @@ void ConsoleCheckDebug()
 #endif
 }
 
-[[nodiscard]] HRESULT ConsoleCreateIoThreadLegacy(_In_ HANDLE Server, const ConsoleArguments* const args)
+HRESULT ConsoleCreateIoThread(_In_ HANDLE Server,

```

Comment:
Adding method comments would be nice for this file.

Reply:
out of scope for this PR.

Reply:
I mean, I know the discussion is like 6 months gone by at this point, but doc comments really _would_ help now that we're gonna be checking this in for real